File: linearsearch.js
## Fun Fact with JS 
function L(l,key){ 
 
  for(i=0;i<=4;i++){
    if(l[i]==key){
      return i;
    }
  }
}
var l = [1,3,5,8,0];
key = 0;
console.log(L(l,key));  #found in index 4

File: Linearsearch.py
### Fun fact with PY
## Define a function 

def Linear(L,key):  
  for i in range(len(L)):
    if L[i] == key:
      return i
  retrun "Not Found"

if __name__ == "__main__":
  list1 = [0,2,1,4,5]
  k = 5
  print(Linear(list,k))  # found in index 4
